server/database-postgres/helpers.js

```diff
@@ -0,0 +1,97 @@
+const axios = require('axios');
+const { db, User, Language, UserLanguage } = require('./models.js');
+
+const getUserById = async (userId) => {
+  try {
+    const user = await User.findOne({ where: { userId } });
+    // console.log('😂 getUserById function result', user.dataValues);
+    return user;
+  } catch (err) {
+    console.error(err);
+    return null;
+  }
+};
+
+const getUserLanguagesByUserId = async (userId) => {
+  try {
+    const languages = await UserLanguage.findAll({ where: { userUserId: userId }})
+      .then(ids => ids.map(id => id.languageLanguageId))
+      .then(languageIds => Language.findAll({ where: {languageId: languageIds}}))
+      .then(languageNames => languageNames.map(lang => lang.name))
+      .catch(error => console.log(error));
+
+      return languages;
+  } catch (err) {
+    console.log(err);
+    return null;
+  }
+}
+
+const getUserNameAndPhoto = async (userId) => {
+  try {
+    const { name, avatarUrl } = await User.findOne({ where: { userId } }, 'name avatarUrl');
+    // console.log('🎷 getUserNameAndPhoto result', name, avatarUrl)
+    return { name, avatarUrl };
+  } catch (err) {
+    return null;
+  }
+};
+
+const getUserSuperhostStatus = async (userId) => {
+  try {
+    const { isSuperhost } = await User.findOne({ where: { userId } }, 'isSuperhost');
+    // console.log('🥁 getUserSuperhostStatus function result', isSuperhost)
+    return { isSuperhost };
+  } catch (err) {
+    return null;
+  }
+};
+
+const formatData = (userData, languageData) => {
+  let formatted = userData;
+  const {
+    userId,
+    name,
+    bio,
+    joinDate,
+    avatarUrl,
+    isSuperhost,
+    identityVerified,
+    responseRate,
+    responseTime
+  } = formatted
```

Comment:
are you using these variables anywhere that you are pulling out of `formatted`?  looks like this function could just be reduces to adding language data onto `userData` obj at `.dataValues.languages` and then return `userData` again

Reply:
will refactor :)